src/vdom/remove.js

```diff
@@ -0,0 +1,3 @@
+export default function remove(el) {
+	el.parentNode && el.parentNode.removeChild(el);
```

Comment:
I'm not really sure, was just making fun of that situation with such conditions. @developit knows more which is faster and which one gzips better. Sorry for confusion!